dev_guide/templates.adoc

```diff
@@ -105,13 +86,77 @@ $ osc process -f examples/sample-app/application-template-dockerbuild.json -v AD
 
 ====
 
-The JSON string can either be redirected to a file or applied directly by piping the process output to the `apply` command, as shown in the following example.
+The JSON string can either be redirected to a file or applied directly by piping the process output to the `create` command:
 
 .Redirecting JSON String
```

Comment:
Changed to "Applying the JSON String", but I'm not convinced that's the best wording...Hard to do it without using "The `create` command".